source/_components/switch.flux.markdown

```diff
@@ -47,4 +48,5 @@ Configuration variables:
 - **sunset_colortemp** (*Optional*): The sun set color temperature. Defaults to `3000`.
 - **stop_colortemp** (*Optional*): The color temperature at the end. Defaults to `1900`.
 - **brightness** (*Optional*): The brightness of the lights. Calculated with `RGB_to_xy` by default.
+- **mode** (*Optional*): select how to pass color temperature to lights defaults to 'xy', kelvin and mired can be selected. (use mired for hue ambiance bulbs)
```

Comment:
Change to:

**mode** (*Optional*): Select how color temperature is passed to lights. Valid values are 'xy', 'mired' and 'kelvin'. Defaults to 'xy'.